compliance_suite/constants.py

```diff
@@ -3,7 +3,7 @@
 SERVICE_INFO_SCHEMA = "service_info.json"
 DRS_ACCESS_SCHEMA= "access_url.json"
 TESTBED_NAME = "DRS Compliance Suite"
-TESTBED_VERSION = "v0.0.0"
+TESTBED_VERSION = "v0.2.0"
```

Comment:
The current testbed version is v1.0.3. Refer to https://github.com/ga4gh/drs-compliance-suite/tags for the latest version.
This change might also require us to update the version in the expected json reports